Remotion/Web/Core/WebString.cs

```diff
@@ -0,0 +1,242 @@
+﻿using System;
+using System.IO;
+using System.Web;
+using System.Web.UI;
+using JetBrains.Annotations;
+using Remotion.Utilities;
+
+namespace Remotion.Web
+{
+  /// <summary>
+  /// The <see cref="WebString"/> structure represents encoded or unencoded HTML strings that are rendered onto a web page.
+  /// Its <see cref="Type"/> determines if and how a <see cref="WebString"/> is encoded before being rendered.
+  /// </summary>
+  public readonly struct WebString : IEquatable<WebString>
+  {
+    private static readonly string[] s_newlineSplitSeparators = { "\r\n", "\r", "\n" };
+
+    /// <summary>
+    /// Creates a <see cref="WebString"/> from the specified <paramref name="html"/>.
+    /// The value will be rendered without being encoded and thus must be a safe string.
+    /// </summary>
+    /// <param name="html">The HTML from which a <see cref="WebString"/> is created. Must not be <see langword="null" />. Must not be <see langword="null" />.</param>
```

Comment:
you're going to update this anyway, but there's two of the "Must not be null" infos.